types/dispatcher.d.ts

```diff
@@ -145,7 +199,7 @@ declare namespace Dispatcher {
     /** Invoked when request is upgraded either due to a `Upgrade` header or `CONNECT` method. */
     onUpgrade?(statusCode: number, headers: string[] | null, socket: Duplex): void;
     /** Invoked when statusCode and headers have been received. May be invoked multiple times due to 1xx informational headers. */
-    onHeaders?(statusCode: number, headers: string[] | null, resume: () => void): boolean;
+    onHeaders?(statusCode: number, headers: Buffer[] | null, resume: () => void): boolean;
```

Comment:
why this change?

Reply:
Because the typescript is wrong. I am merely correcting the type definition file.

To be honest, this is why I hate javascript libraries, nobody knows what the correct types are anywhere.